tooling/reproducible/linux_repro_build_compare.sh

```diff
@@ -84,8 +84,116 @@ setEnvironment() {
   ls -ld "$CC" "$CXX" "/usr/lib/jvm/jdk-${BOOTJDK_VERSION}/bin/javac" || exit 1
 }
 
+# Function to check if a value is in the array
+containsElement () {
+  local e
+  for e in "${@:2}"; do
+    if [ "$e" = "$1" ]; then
+      return 0  # Match found
+    fi
+  done
+  return 1  # No match found
+}
+
+setBuildArgs() {
+  local CONFIG_ARGS=("--disable-warnings-as-errors" "--enable-dtrace" "--without-version-pre" "--without-version-opt" "--with-version-opt")
+  local NOTUSE_ARGS=("--configure-args" "--enable-sbom-strace")
```

Comment:
--configure-args should not be being ignored, only --enable-sbom-strace

Reply:
--configure-args  is not ignored. Its basically using the same logic as window repro_build code. it was added back `final_params="$build_string --configure-args \"$config_string\" $jdk"`. The idea is do with --configure-args differently as it need the double quoted.

Reply:
Yes, you should not need this if we fix the SBOM --configure-args BUILD_ARG, which is currently missing the double-quotes
The problem here is we are adding assumptions about what configure args are expected, we should just pass them through.
The Windows logic is wrong too.